tests/plugins/formatting.test.ts

```diff
@@ -448,6 +448,22 @@ describe("Autoresize", () => {
     expect(model.getters.getColSize(sheetId, 0)).toBe(initCellWidth);
   });
 
+  test("Can autoresize a column with multiline content", () => {
+    const content = `Hello this is \nmultiline content for test`;
+    setCellContent(model, "A1", content);
+    model.dispatch("AUTORESIZE_COLUMNS", { sheetId, cols: [0] });
+    const cartesian = toCartesian("A1");
+    const position = {
+      sheetId: sheetId,
+      col: cartesian.col,
+      row: cartesian.row,
+    };
+    const style = model.getters.getCellComputedStyle(position);
+    expect(model.getters.getColSize(sheetId, 0)).toBe(
+      model.getters.getTextWidth("multiline content for test", style) + hPadding
```

Comment:
it would be better to recompute the size, the same way it is done in the plugin ( with the `Math.max(...`) this